src/routes/guest.js

```diff
@@ -1,12 +1,18 @@
 import express from 'express';
 import { newGuest } from '../controllers/guest-controller';
+import '../controllers/application-controller';
+import { isSetup } from '../controllers/application-controller';
 
 // eslint-disable-next-line new-cap
 const router = express.Router();
 
-router.get('/', (req, res, next) =>
-  res.render('guests', { layout: 'default', template: 'guess-template' })
-);
+router.get('/', async (req, res, next) => {
```

Comment:
Esto no va así, lo que queremos no es que los lleve a first-time setup, lo que queremos es que si se meten con intención, sea porque no se ha hecho, por lo que los is setups van en las rutas de onboarding